packages/vue/src/components/atoms/SfCircleIcon/SfCircleIcon.stories.js

```diff
@@ -1,68 +1,68 @@
 // /* eslint-disable import/no-extraneous-dependencies */
-import { storiesOf } from "@storybook/vue";
+import { storiesOf } from '@storybook/vue';
```

Comment:
The `'` in this file (instead of `"`) look too like there is something wrong with your linter setup. Please make sure there is no global linter config on your machine which overrides project-specific settings. ESLint + Prettier are enough for this project, all other linters/formatting plugins should be disabled. If you use WebStorm, enable both ESLint and Prettier auto-detection in your settings; if you use VS Code, enable the prettier plugin.
As lint-on-commit is disabled at the moment, you should manually run prettier via your IDE for this file.

Otherwise it will be very hard to resolve the merge conflict... 🙉💣

Reply:
I have prettier enabled and I'm pretty sure there are no global linters that could override the project specific ones. I guess that `'` is leftover from the branch that I've checkout from. And the SVG problem happens, when I'm switching to develop, really hard to say what causes the issue, but its truly annoying :smile: All I've changed is aria labels :woman_shrugging:

Reply:
Checked the file - its a leftover from the base brach, my IDE is working according to the specific settings :)